### DISCUSSION TITLE
Allow omitting `baseUrl` in tsconfig

### DISCUSSION THREAD

**Discussion by itsMapleLeaf** [remix-run/remix#5578] [Proposals]
[Since version 4.1, `paths` can be used without specifying `baseUrl`.](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1/#paths-without-baseurl) Omitting `baseUrl` is useful to keep from accidentally auto-importing from the project root. Currently, Remix forces baseUrl to be set. It should update its defaults:
```diff
-   "baseUrl": ".",
    "paths": {
      "~/*": ["./app/*"]
    },
```

**Comment by aaronadamsCA** [comment]
We are not configuring `paths["~/*"]` in our project at all, which makes it extra annoying that `remix watch` keeps modifying our `tsconfig.json`. I can't say I understand why `baseUrl` would be considered "mandatory."
